package.json

```diff
@@ -9,8 +9,8 @@
         "prettier:fix": "pnpm prettier . --write",
```

Comment:
while we're at it, perhaps we can also update format and lints as well?
```suggestion
        "format": "prettier . --write",
```

Reply:
good idea!

Reply:
Sounds smart. How would you proceed with the other commands?

```bash
"prettier:check": "pnpm prettier . --check",  # format:check
"prettier:fix": "pnpm prettier . --write", # format
"eslint:check": "pnpm eslint .", # lint
"eslint:fix": "pnpm eslint --fix .", # lint:check
```